pkg/cmd/migrate.go

```diff
@@ -23,6 +25,7 @@ import (
 func RegisterMigrateFlags(cmd *cobra.Command) {
 	cmd.Flags().String("datastore-engine", "memory", fmt.Sprintf(`type of datastore to initialize (%s)`, datastore.EngineOptions()))
 	cmd.Flags().String("datastore-conn-uri", "", `connection string used by remote datastores (e.g. "postgres://postgres:password@localhost:5432/spicedb")`)
+	cmd.Flags().Bool("datastore-aws-iam-auth", false, "set to true to enable AWS IAM authentication (postgres driver only)")
```

Comment:
Ideally this would be `--datastore-credentials-provider-name` and then custom fields for the chosen provider (for now, just `aws`)